pkgs/applications/video/mpv/default.nix

```diff
@@ -82,7 +82,20 @@ let
   inherit (darwin.apple_sdk_11_0.frameworks)
     AVFoundation CoreFoundation CoreMedia Cocoa CoreAudio MediaPlayer Accelerate;
   luaEnv = lua.withPackages (ps: with ps; [ luasocket ]);
-in stdenv.mkDerivation (finalAttrs: {
+
+  overrideSDK = platform: version:
+    platform // lib.optionalAttrs (platform ? darwinMinVersion) {
+      darwinMinVersion = version;
+    };
+
+  stdenv' = if !(swiftSupport && stdenv.isDarwin && stdenv.isx86_64)
```

Comment:
This is general code style: 

It is better to write 

```nix
if cond then branch1 else branch2
```

Instead of

```python
if !(cond) then branch2 else branch1
```

Reply:
I did that to put the trivial branch first, but I can switch it to the general code style.

Reply:
I pushed an updated PR with the logic corrected.